src/sentry/search/snuba/executors.py

```diff
@@ -750,11 +750,6 @@ def query(
 
         if not end:
             end = now + ALLOWED_FUTURE_DELTA
-            allow_postgres_only_search = True
```

Comment:
This piece part of the code was likely running for most searches which sets the feature to on anyways. I added some metrics to check the number of searches that use postgres_only vs non-postgres only and the request distribution was unchanged when I enabled the feature flag for all orgs.